wikimedia/wikipedia-ios
# Watchlist - Push user to diff when tapping watchlist cell

**Phabricator:** https://phabricator.wikimedia.org/T335606 (partially)

### Notes
Pushes the user to the diff comparison view when they tap a watchlist cell. 

**Dependent on https://github.com/wikimedia/wikipedia-ios/pull/4623 being merged first**.

### Test Steps
1. Open Watchlist
2. Tap a revision cell
3. Confirm you're pushed to the diff comparison view
